src/app/dto/AdminDTO/admin.deck.with.pages.DTO.ts

```diff
@@ -0,0 +1,11 @@
+import {AdminDeck} from "./admin.deck.DTO";
+
+export class AdminDeckByPage {
```

Comment:
This class is the same as the previous one and the next one.
TypeScript supports generics.
Lets refactor it later.